Formula/trojan-go.rb

```diff
@@ -0,0 +1,114 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go.git",
+      tag:      "v0.10.0",
+      revision: "a9ce6d4ada9143d58e8d1373d23f9ebfe871119e"
+  license "GPL-3.0-only"
+  head "https://github.com/p4gefau1t/trojan-go.git"
+
+  depends_on "go" => :build
+
+  resource "geoip" do
+    geoip_hash_url = "https://github.com/v2fly/geoip/releases/latest/download/geoip.dat.sha256sum"
+    url "https://github.com/v2fly/geoip/releases/latest/download/geoip.dat"
+    sha256 Utils.safe_popen_read("curl", "-L", geoip_hash_url)[0, 64]
+  end
+
+  resource "geosite" do
+    geosite_hash_url = "https://github.com/v2fly/domain-list-community/releases/latest/download/dlc.dat.sha256sum"
+    url "https://github.com/v2fly/domain-list-community/releases/latest/download/dlc.dat"
+    sha256 Utils.safe_popen_read("curl", "-L", geosite_hash_url)[0, 64]
+  end
+
+  def install
+    commit = Utils.git_head
+    tag = Utils.safe_popen_read("git", "tag", "--contains", "HEAD").strip
```

Comment:
Is this different from the version of the formula? If it isn't, you can just drop this line entirely and do
```
-X github.com/p4gefau1t/trojan-go/constant.Version=#{version}
```

Reply:
Resolved.